docker/builder/builder.go

```diff
@@ -52,9 +54,30 @@ func (d *DaemonBuilder) Build(ctx context.Context, imageName string) error {
 		return err
 	}
 	defer buildCtx.Close()
+	var l logger.Logger
+	if d.LoggerFactory != nil {
+		l = d.LoggerFactory.CreateBuildLogger(imageName)
+	}
 
 	var progBuff io.Writer = os.Stdout
```

Comment:
I wonder about that : Should we make the assumption that the loggerFactory is not nil, and/or have a default one that just *eats* the output ? This would remove the need to check for `nil` in multiple places here.